pkg/virt-operator/resource/generate/components/deployments.go

```diff
@@ -40,7 +40,11 @@ const (
 
 	VirtAPIName        = "virt-api"
 	VirtControllerName = "virt-controller"
-	VirtOperatorName   = "virt-operator"
+	VirtOperatorName   = "virt-operator"	
+	PrometheusUrl	   = "prometheus.kubevirt.io"
+	KubeVirtUrl	   = "kubevirt.io"
```

Comment:
Consider using our `api.AppLabel` constant, like it is for instance used on the services in this file already.